Makefile

```diff
@@ -13,14 +13,14 @@ include Makefile.global
 all: extension
 
 # build extension
-extension:
+extension: $(citus_abs_srcdir)/src/include/citus_version.h
```

Comment:
Why add a new header at all? What's wrong with using the existing `citus_config.h` one? PostgreSQL just uses `pg_config.h` for what they need.